README.md

```diff
@@ -7,6 +7,8 @@ The HMS Unity plugin helps you integrate all the power of Huawei Mobile Services
 * Ads: Interstitial, rewarded videos and banner
```

Comment:
Is Kotlin-stdlib needed? is an extra 1.32MB

Reply:
unfortunately it's necessary for initialization.

Reply:
Why is this a dependency? Our SDKs use Kotlin?

Reply:
![kotlib](https://user-images.githubusercontent.com/8115505/97303909-12731a80-186c-11eb-9ae8-b995e64ab4f1.png)

Reply:
Analytics kit need agconnect credential library that library using Kotlin